### PR TITLE
[NUI] Update Animation documents

### PR DESCRIPTION
**Pull request by rabbitfor**
Signed-off-by: Jiyun Yang <ji.yang@samsung.com>

### Change Description ###

- Fix examples that are not valid any more.
- Improve image resources
- Remove contents that are too deep in technically.

### CODE REVIEW

File: docs/application/dotnet/guides/nui/animation.md [hunk 1]
```diff
@@ -2,220 +2,187 @@
...
-    }
-
-    AlphaFunction af(alphafunc);
-    animation.SetDefaultAlphaFunction(af);
-    ```
+### Built-in Alpha Function
 
-- By using delegates:
+The simplest way to define an alpha function is to use `built-in alpha function` which NUI provides for the convenience.
+See [Tizen.NUI.AlphaFunction.BuiltinFunctions](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.AlphaFunction.BuiltinFunctions.html), to check a supporting list.
```
[the remark this answers is not in the document]
Comment (rabbitfor) [reply docs/application/dotnet/guides/nui/animation.md]: What I meant was not about "how to check", so I rephrase it to

For more information on a supporting list, see [Tizen.NUI.AlphaFunction.BuiltinFunctions](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.AlphaFunction.BuiltinFunctions.html).

**Comment by rabbitfor** [comment]
@divyk Done fixing as your comments

File: docs/application/dotnet/guides/nui/animation.md [hunk 2]
```diff
@@ -1,228 +1,195 @@
...
-1.  Process the message and call the `SetPosition()` method.
-2.  Apply the animation and call the `SetPosition()` method.
-3.  Render the frame.
+<div style="text-align:center;width:100%;"><img src="./media/bezier.svg" /></div>
 
 <a name="sample"></a>
 ## Sample Animation
 
-You can use an `animation-hello-world.cs` sample to see in practice how the `AnimateBy()` and `AnimateTo()` methods, and alpha functions work.
+You can use an `animation-hello-world.cs` sample to practice and understand how the `AnimateBy()` and the `AnimateTo()` methods, and alpha functions work.
```
Comment (mijin85cho) [inline docs/application/dotnet/guides/nui/animation.md]: Question. Isn't the sample name "animation-hello-world-tutorial.cs" according to line 194?
The sample name described here is different.

File: docs/application/dotnet/guides/nui/animation-types.md [hunk 3]
```diff
@@ -1,127 +1,45 @@
...
-// Black points
-Position position0 = new Position(200.0f, 200.0f, 0.0f);
-Position position1 = new Position(300.0f, 300.0f, 0.0f);
-Position position2 = new Position(400.0f, 400.0f, 0.0f);
 
-Path path = new Path();
-path.AddPoint(position0);
-path.AddPoint(position1);
-path.AddPoint(position2);
+To animate `View` along this path, use the following function:
```
Comment (mijin85cho) [inline docs/application/dotnet/guides/nui/animation-types.md]: Question. In the sample code, `view` is used.
However in this line `View` is described.
Don't we need to use small letter for `View`?

File: docs/application/dotnet/guides/nui/animation.md [hunk 4]
```diff
@@ -1,228 +1,195 @@
...
 
-![NUI class hierarchy](media/NUI_Class_Hierarchy.png)
+Some view properties are `animatable`, such as `Color`, `Scale`, `Orientation`, and `Position`.
 
--   The `Tizen.NUI.Animatable` class contains methods related to animation properties, such as the `GetProperty()` and `IsPropertyAnimatable()` methods.
--   The `Tizen.NUI.Animation` class contains animation methods, such as the `AnimateBy()` and `AnimateTo()` methods.
+<div style="text-align:center;width:100%;"><img src="./media/properties.svg" /></div>
 
-To implement a basic animation, specify an animation object that takes place in milliseconds:
+For more information on the animatable properties, see the `Remarks` section in [Tizen.NUI.BaseComponents.View](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html).
```
[the remark this answers is not in the document]
Comment (divyk) [reply docs/application/dotnet/guides/nui/animation.md]: @mijin85cho , thanks, yes I agree. How about giving it as the following:

For more information on the animatable properties, see the `Remarks` section of each property in [Tizen.NUI.BaseComponents.View](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html). Note that, **Remarks** sections are sub-item of each property.

File: docs/application/dotnet/guides/nui/animation.md [hunk 5]
```diff
@@ -1,228 +1,195 @@
...
-_animation.AnimateTo(_text, "Orientation", new Rotation(new Radian(new Degree(0.0f)), PositionAxis.X), 500, 1000, new AlphaFunction(AlphaFunction.BuiltinFunctions.EaseInOutSine));
+animation.AnimateBy(view2, "PositionY", 100);
 
-_animation.AnimateBy(_text, "ScaleX", 3, 1000, 1500);
-_animation.AnimateBy(_text, "ScaleY", 4.0f, 1500, 2000);
+animation.Play();
 ```
 
-You can also pass properties to an animation method through the `Property` class instantiation:
+### Keyframe Animation
```
[the remark this answers is not in the document]
Comment (divyk) [reply docs/application/dotnet/guides/nui/animation.md]: @mijin85cho , I agree.

File: docs/application/dotnet/guides/nui/animation.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/application/dotnet/guides/nui/animation.md]: @divyk I think adding the first sentence only would be enough. Only below one.

For more information on the animatable properties, see the `Remarks` section of each property in [Tizen.NUI.BaseComponents.View](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html).

[the remark this answers is not in the document]
Comment (divyk) [reply docs/application/dotnet/guides/nui/animation.md]: @mijin85cho , I agree, thanks.

File: docs/application/dotnet/guides/nui/animation.md [hunk 6]
```diff
@@ -1,228 +1,195 @@
...
+You can use animations to allow objects to move around and change their properties for a specified duration. NUI implements an easy-to-use animation framework that allows you to create visually rich applications. The `Tizen.NUI.Animation` class can be used to animate the [animatable properties](#animatableproperties) for any number of `View` objects.
 
-NUI animations [occur in a dedicated thread](#multithreading). This allows animations to run smoothly, regardless of the time taken to process the input, events, and other factors in the application code.
+NUI animations occur in a dedicated thread. This allows animations to run smoothly, regardless of the time taken to process the input, events, and other factors in the application code.
 
-The figure shows the NUI class hierarchy:
 
-**Figure: NUI class hierarchy**
+<a name="animationproperties"></a>
+## Animation Properties
```
[the remark this answers is not in the document]
Comment (rabbitfor) [reply docs/application/dotnet/guides/nui/animation.md]: This section is about, `animatable properties` and `how to apply animation to them`.
I'm not sure which one is better.  `Animatable Properties` and `Animation Properties`

File: docs/application/dotnet/guides/nui/animation-types.md (hunk 3 above)
Comment (rabbitfor) [reply docs/application/dotnet/guides/nui/animation-types.md]: `View` is a class name of animation target,
and `view` in the sample code is an example instance of `View` class.

Animation target is always `View` class and its instance can have any name such as `view`, `imageView` and `videoView`, so I used `View` in the description.

File: docs/application/dotnet/guides/nui/animation.md [hunk 7]
```diff
@@ -1,228 +1,195 @@
 # Animation
 
-You can use animations to allow objects to move around and change their properties for a specified duration. NUI implements an easy-to-use animation framework that allows you to create visually rich applications. The `Tizen.NUI.Animation` class can be used to animate the [animatable properties](#animatableproperties) for any number of objects, typically view object.
+You can use animations to allow objects to move around and change their properties for a specified duration. NUI implements an easy-to-use animation framework that allows you to create visually rich applications. The `Tizen.NUI.Animation` class can be used to animate the [animatable properties](#animatableproperties) for any number of `View` objects.
```
[the remark this answers is not in the document]
Comment (rabbitfor) [reply docs/application/dotnet/guides/nui/animation.md]: Currently, all NUI `Animation` APIs supports only `View` object, so the fixed description is right.

And I've fixed broken link.

File: docs/application/dotnet/guides/nui/animation.md [hunk 8]
```diff
@@ -1,228 +1,195 @@
...
+NUI animations occur in a dedicated thread. This allows animations to run smoothly, regardless of the time taken to process the input, events, and other factors in the application code.
 
-The figure shows the NUI class hierarchy:
 
-**Figure: NUI class hierarchy**
+<a name="animationproperties"></a>
+## Animation Properties
 
-![NUI class hierarchy](media/NUI_Class_Hierarchy.png)
+Some view properties are `animatable`, such as `Color`, `Scale`, `Orientation`, and `Position`.
```
[the remark this answers is not in the document]
Comment (rabbitfor) [reply docs/application/dotnet/guides/nui/animation.md]: @mijin85cho  As your comment, I found `Color` is not public API so I changed it to `Opacity`.
And also changed text in the picture to sync with descriptions.

In description:  `Opacity`, `Scale`, `Orientation`, `Position`
In the picture: `Opacity`, `Scale`, `Orientation (ZAxis)`, `Position`

Now the only difference is `Orientation`.
The picture shows `Orientation` animation with z-axis value.
`Orientation` is confusing concept in animation so I wanted to add `(ZAxis)` to make it easy to understand what the picture tries to show.

**Comment by rabbitfor** [comment]
@mijin85cho Thank you for the detailed review.
I have updated code as your comment, and made comments to some of them.
Please have a look.

File: docs/application/dotnet/guides/nui/animation-types.md [hunk 9]
```diff
@@ -2,15 +2,15 @@
 
 ## Path Animation
 
-The `Path` class can be used to animate the position and orientation of actors. (a three-dimentional parametric curve)
+The `Path` class is three-dimensional parametric curve that is used to animate the position and orientation of actors.
```
Comment (divyk) [inline docs/application/dotnet/guides/nui/animation-types.md]: There is a just a missing article 'a', please make the small change:

The `Path` class is a three-dimensional parametric curve that is used to animate the position and orientation of actors.

File: docs/application/dotnet/guides/nui/animation-types.md [hunk 10]
```diff
@@ -2,15 +2,15 @@
...
 
 The `Path` consists of several points and each adjacent pair of points becomes a segment of the path. Each segment can be curved by adding control points to it.
 
 In the following example, the P1, P2 and P3 form the path where the blue circle travels. The C1 and C2 are the control points for a first segment `[P1-P2]`, where C3 and C4 are for a second segment `[P2-P3]`. The control points express the curvature of the path.
 
 <div style="text-align:center;width:100%;"><img src="./media/path.svg" /></div>
 
-The code would look like:
+To generate `Path' object, use the following code:
```
Comment (divyk) [inline docs/application/dotnet/guides/nui/animation-types.md]: There is a just a missing article 'the', please make the small change:

To generate the `Path' object, use the following code:

File: docs/application/dotnet/guides/nui/animation.md [hunk 11]
```diff
@@ -123,27 +123,27 @@ During the playback, you can receive notifications at various stages of the anim
...
 
 <a name="alphafunctions"></a>
 ## Alpha Functions
 
-In animations, alpha functions are used to specify the an animation progresses over time. This allows the animation to be, for example, accelerated, decelerated, repeated, or bounced.
+In animations, alpha functions are used to specify an animation progresses over time. The alpha function allows the animation to be accelerated, decelerated, repeated, or bounced.
 
 ### Built-in Alpha Function
 
 The simplest way to define an alpha function is to use `built-in alpha function` which NUI provides for the convenience.
```
Comment (divyk) [inline docs/application/dotnet/guides/nui/animation.md]: Please make this small change **which** to **that**:

The simplest way to define an alpha function is to use `built-in alpha function` that NUI provides for the convenience.

@prasadam-p, I have one more query, is there any specific reason to call out `built-in alpha function` in code tag? Because I see the ENUM value is mentioned differently, and so is in the code. Please clarify my doubt.

File: docs/application/dotnet/guides/nui/animation.md [hunk 12]
```diff
@@ -53,10 +53,10 @@ animation.AnimateBy(view2, "PositionY", 100);
 animation.Play();
 ```
 
-### KeyFrame Animation
-NUI provides support for animating between several different values for a View. This can be done by defining the `KeyFrames`.
+### KeyFrames Animation
+By defining the `KeyFrames` object, NUI supports animation between several different values for a View.
```
Comment (divyk) [inline docs/application/dotnet/guides/nui/animation.md]: Please make the View in code tag as follows:

By defining the `KeyFrames` object, NUI supports animation between several different values for a `View`.

File: docs/application/dotnet/guides/nui/animation.md (hunk 11 above)
Comment (rabbitfor) [reply docs/application/dotnet/guides/nui/animation.md]: I removed code tag from `built-in alpha function`

**Comment by rabbitfor** [comment]
@divyk Thank you for the review. Please check the updated code.

File: docs/application/dotnet/guides/nui/animation-types.md [hunk 13]
```diff
@@ -1,127 +1,45 @@
...
+The `Path` consists of several points and each adjacent pair of points becomes a segment of the path. Each segment can be curved by adding control points to it.
 
-**Figure: Path animation**
+In the following example, the P1, P2 and P3 form the path where the blue circle travels. The C1 and C2 are the control points for a first segment `[P1-P2]`, where C3 and C4 are for a second segment `[P2-P3]`. The control points express the curvature of the path.
 
-![Path animation](./media/path_animation.png)
+<div style="text-align:center;width:100%;"><img src="./media/path.svg" /></div>
 
-The following code presents the black points:
+To generate the `Path' object, use the following code:
```
Comment (mijin85cho) [inline docs/application/dotnet/guides/nui/animation-types.md]: Just one more check before merging. 
@divyk , `Path' is not in code tag style and just quotation marks are used. Is it intended?

image

Comment (divyk) [reply docs/application/dotnet/guides/nui/animation-types.md]: @mijin85cho , thanks I missed seeing it.

@rabbitfor , please make this small change:

To generate the `Path` object, use the following code:

**Comment by rabbitfor** [comment]
Sorry my mistake. I uploaded fixed one. Thank you for your help.

**Comment by rabbitfor** [comment]
@mijin85cho @divyk Sorry, I made mistakes a lot. Please check updated code. Thank you for your help.
